docs/docs/features/splits.mdx

```diff
@@ -0,0 +1,162 @@
+
+{/* Here is an example diagram of a split keyboard with a dongle:
+
+```mermaid
+graph TD;
+    A[Computer/Host Device] <-- USB/BLE --> DONGLE[Dongle/Central];
+    DONGLE -. BLE .-> LEFT[Left Half/Peripheral];
+    DONGLE -. BLE .-> RIGHT[Right Half/Peripheral];
+```
+
+Dotted lines represent BLE connections.
+Solid lines represent USB connections. */}
+
+:::info
+For more information on how to set up a two split keyboard, refer to [New Keyboard Shield](../development/new-shield.mdx) under Guides.
+After setting up the split keyboard, you can follow the steps below to add a dongle to the setup.
+:::
+
+## Benefits/Disadvantages of using a Dongle/Split setup
+
+Benefits:
+
+- Longer battery life for peripheral halves of the keyboard.
+- Dongle can be powered by the host device and no long wires are needed.
+- Easier to connect to the host device.
+- Only the central side needs to reflashed for keymap updates
+
+Disadvantages:
+
+- More complex setup.
+- More components needed.
+- Added latency due to the extra hop between the host device and the keyboard halves.
+- Each half of the keyboard needs its own firmware (ex: left, right, and dongle).
+
+:::warning[Reset/Bootloader behavior]
+When using a combo to trigger reset/bootloader, only the central side will be affected. To avoid this, you can avoid using the combo and instead use a physical reset button on the central half.
+That will allow you to locally reset the central/peripheral side without affecting other halves. 
+Further explanation can be found under [Behavior Reset#split-keyboards](../behaviors/reset.md#split-keyboards)
+:::
+
+## Building a Split Keyboard shield with a Dongle
+
+:::note
+Because we will be changing the roles of the keyboard halves, we will also need to flash the reset firmware to the keyboard halves.
+Information about the reset firmware can be found under Troubleshooting [Reset firmware](../troubleshooting/connection-issues.mdx#acquiring-a-reset-uf2)
+:::
+
+### Kconfig.shield
+
+First we will introduce a third split to our keyboard configuration. This will be used as the dongle.
+
+```kconfig
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD_LEFT
+    def_bool $(shields_list_contains,my_board_left)
+
+config SHIELD_MY_BOARD_RIGHT
+    def_bool $(shields_list_contains,my_board_right)
+```
+
+### Kconfig.defconfig
+
+Next we will define the roles of the keyboard halves. The left and right halves will be set as peripherals, and the dongle will be set as the central.
+We will also only give the dongle the keyboard name.
+
+```kconfig
+if SHIELD_MY_BOARD_DONGLE
+
+config ZMK_KEYBOARD_NAME
+    default "My Board"
+
+config ZMK_SPLIT_ROLE_CENTRAL
+    default y
+
+endif
+
+if SHIELD_MY_BOARD_DONGLE || SHIELD_MY_BOARD_LEFT || SHIELD_MY_BOARD_RIGHT
+
+config ZMK_SPLIT
+    default y
+
+endif
+```
```

Comment:
I think it would be nice to recommend a setup that makes it easy to switch between dongle-enabled and dongle-disabled setups. I'm pretty sure you could write this file so that a single `CONFIG_ZMK_SPLIT_ROLE_CENTRAL=n` in `shield_left.conf` would cause the left half to be built as peripheral, while not having that line has it built as a dongle.

Then keyboard designers could include the files necessary for dongles, and users could choose whether to build with or without dongle.

Reply:
CONFIG_ZMK_SPLIT_ROLE_CENTRAL=n wouldnt that make it so it is not a central?

Reply:
Moved config around to support still building left side as central